### DISCUSSION TITLE
Moving applications between projects.

### DISCUSSION THREAD

**Discussion by sulliwan** [zitadel/zitadel#8841] [Ideas]
So I started using Zitadel and of course did not read the manual before creating my first application, so I created it in the default ZITADEL project :(

Ok, no big deal, I'll just create a real project and move the application, I thought. Turns out, it's not possible to do that, at least not via any method I was able to discover. Sure, I could remove the current application, create a new one and change the id/secret on the service side, but as it turns out, the service the application is connected to (Tailscale) doesn't have any means to easily change the key or secret either.

I wonder if this is a common enough problem that a way of moving applications would make sense to have?

**Comment by hifabienne** [comment]
So far we didn't hear about that need.
As it is not a big thing to just create a new application with the correct information.
Is it possible to create a new service in tailscale?
